arangod/GeneralServer/RestHandler.cpp

```diff
@@ -563,8 +563,8 @@ void RestHandler::compressResponse() {
 /// @brief generates an error
 ////////////////////////////////////////////////////////////////////////////////
 
-void RestHandler::generateError(rest::ResponseCode code, int errorNumber) {
-  char const* message = TRI_errno_string(errorNumber);
+void RestHandler::generateError(rest::ResponseCode code, ErrorCode errorNumber) {
+  const auto* message = TRI_errno_string(errorNumber);
```

Comment:
I personally find `char const*` more informative than a generic `const auto*`, but this is obviously opinionated.
```suggestion
  char const* message = TRI_errno_string(errorNumber);
```